packages/agoric-cli/src/commands/perf.js

```diff
@@ -76,9 +74,10 @@ export const makePerfCommand = async logger => {
             if (status.error) {
               console.error(status.error);
               exit(1);
-            } else if (status.numWantsSatisfied)
+            } else if (status.numWantsSatisfied) {
               process.stdout.write(`satisfied: ${status.numWantsSatisfied}\n`);
-            exit(0);
```

Comment:
this would exit whenever a non-error update arrived. in practice, this happens about the same time as numWantsSatisfied.